spec/ffprober/ffmpeg/finder_spec.rb

```diff
@@ -1,11 +1,10 @@
-# encoding: utf-8
 require 'spec_helper'
 
-describe Ffprober do
+describe Ffprober::Ffmpeg::Finder do
 
   describe 'if no ffprobe is found' do
     it 'should raise a exception if there is no ffmpeg' do
-      allow(Ffprober).to receive(:path).and_return('nonexistant')
+      allow(described_class).to receive(:path).and_return('nonexistant')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.